recipes-nemomobile/cor/cor_git.bb

```diff
@@ -1,9 +1,9 @@
 SUMMARY = "Nemomobile's Cor"
-HOMEPAGE = "https://git.merproject.org/mer-core/cor"
+HOMEPAGE = "https://github.com/sailfishos/cor"
 LICENSE = "LGPL-2.1+"
 LIC_FILES_CHKSUM = "file://COPYING;md5=d2c621b4dd653efb7cbad641e0a30c59"
 
-SRC_URI = "git://git.merproject.org/mer-core/cor.git;protocol=https \
+SRC_URI = "git://github.com/sailfishos//cor.git;protocol=https \
```

Comment:
One `/` too many

Reply:
This still needs patches to successfully create `qtscenegraph-adaptation`